lib/pdf-editor-view.js

```diff
@@ -29,6 +29,7 @@ export default class PdfEditorView extends ScrollView {
     this.currentScale = 1.5;
     this.defaultScale = 1.5;
     this.scaleFactor = 10.0;
+    this.fitToWidth = atom.config.get('pdf-view.fitToWidth')
```

Comment:
This should be changed to `atom.config.get('pdf-view.fitToWidthOnOpen')`, right?

Also, how do you feel about calling the variable `fitToWidthOnOpen` as well, so that it's clearer in the code what information is stored in it and how it should be used?